SynCrtSock.pas

```diff
@@ -1901,8 +1913,8 @@   THttpServer = class(THttpServerGeneric)
     // - expects the port to be specified as string, e.g. '1234'; you can
     // optionally specify a server address to bind to, e.g. '1.2.3.4:1234'
     // - you can specify a number of threads to be initialized to handle
-    // incoming connections (default is 32, which may be sufficient for most
-    // cases, maximum is 256) - if you set 0, the thread pool will be disabled
+    // incoming connections. Default is 32, which may be sufficient for most

```

Comment:
I remove ( ) from comment because Lazarus parser not work correctly with ( ). Ctrl+Shift+Down not work (